README.md

```diff
@@ -2,13 +2,13 @@
 
 # Essencium
 
-[![Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml/badge.svg)](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
+[Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml)
```

Comment:
This is supposed to be a status badge of our pipeline. It is corrupt due to the fact that we changed the pipeline workflow name from `build` to `ci` a couple of months ago. The correct path is:

```suggestion
[![Build](https://github.com/Frachtwerk/essencium-frontend/actions/workflows/build.yml/badge.svg)]
```